dev-lang/ghc/ghc-9.0.2-r2.ebuild

```diff
@@ -80,25 +76,18 @@ BUMP_LIBRARIES=(
 
 LICENSE="BSD"
 SLOT="0/${PV}"
-KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
-IUSE="big-endian +doc elfutils ghcbootstrap ghcmakebinary +gmp llvm numa profile test unregisterised"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="big-endian +doc elfutils ghcbootstrap ghcmakebinary +gmp numa profile test"
 IUSE+=" binary"
 RESTRICT="!test? ( test )"
 
-LLVM_MAX_SLOT="13"
```

Comment:
hm, why did the LLVM stuff get lost?

Reply:
ah. This would be because our ::haskell and ::gentoo ebuilds are not in sync. We ideally need to be committing to ::haskell and then syncing to ::gentoo (or at least if someone opens a PR in ::gentoo they need to do that in ::haskell as well).

As I synced from ::haskell this got lost. I'll fix it up.